.gitignore

```diff
@@ -7,4 +7,5 @@
 /.pypirc
 /.tox/
 .dropbox
-Icon
\ No newline at end of file
+Icon
+/pytestdebug.log
```

Comment:
what's this for?

Reply:
I believe this file is created when you run pytest --debug